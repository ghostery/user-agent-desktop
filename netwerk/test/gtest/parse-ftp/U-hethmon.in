ftp> open ftp.hethmon.com
Connected to ftp.hethmon.com.
220 Hethmon Brothers FTP Server for OS/2 ** BETA 2.0 **. Ready for new user.
Name (ftp.hethmon.com:cyp): 
331 Guest logins allowed. Email address required for anonymous.
230 User foo@bar.com logged in.
Remote system type is UNIX.
ftp> cd pub
250 Directory changed to  "/pub"
ftp> ls
200 Port command ok
150 Opening ASCII mode data connection.
d--x------ 1 ftp ftp          5 Mar 21  1999 ..
d--x------ 1 ftp ftp          0 Feb 16  2002 beta
---------- 1 ftp ftp      35573 May  5  2000 CheckQ2.zip
---------- 1 ftp ftp    1550521 May  5  2000 Ftpd106.zip
---------- 1 ftp ftp      35459 Feb  7  2001 hmailbox.zip
---------- 1 ftp ftp      55463 May  5  2000 HRxMail.zip
---------- 1 ftp ftp      34950 May  5  2000 HRxPass.zip
---------- 1 ftp ftp    1320184 May  5  2000 inetmail-1.2.1.pro.zip
---------- 1 ftp ftp    1218483 May  5  2000 inetmail-1.2.1.zip
---------- 1 ftp ftp    1615428 May  5  2000 inetmail-1.3.0.pro.zip
---------- 1 ftp ftp    1425567 May  5  2000 inetmail-1.3.0.zip
---------- 1 ftp ftp    1629145 Jan  9  2001 inetmail-1.3.18.zip
---------- 1 ftp ftp    1625314 Feb  7  2001 inetmail-1.3.18a.zip
---------- 1 ftp ftp    1407069 May  5  2000 inetmail-1.5.0.pro.zip
---------- 1 ftp ftp    1470338 Sep 27  2000 inetmail-1.5.31.pro.zip
---------- 1 ftp ftp    1583435 Jan  4  2001 inetmail-1.5.32.pro.zip
---------- 1 ftp ftp    1429364 May  5  2000 inetmail-1.5.5.pro.zip
---------- 1 ftp ftp    1471373 May  5  2000 inetmail-1.5.6.pro.zip
d--x------ 1 ftp ftp          0 Feb  8  2001 misc
---------- 1 ftp ftp      30913 May  5  2000 MXLookup.zip
d--x------ 1 ftp ftp          0 May  2  2002 perseus
d--x------ 1 ftp ftp          0 Feb  8  2001 rfcs
d--x------ 1 ftp ftp          0 Feb  8  2001 scripts
d--x------ 1 ftp ftp          0 Feb  8  2001 tcpip
d--x------ 1 ftp ftp          0 May  5  2000 testcase
d--x------ 1 ftp ftp          0 Feb  7  2001 tools
226 Data transfer complete
ftp> close
221 Service closing control connection.
ftp> 