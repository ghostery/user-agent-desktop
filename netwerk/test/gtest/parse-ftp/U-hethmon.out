<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

03-21-1999  00:00:00       <DIR>          ..
02-16-2002  00:00:00       <DIR>          beta
05-05-2000  00:00:00                35573 CheckQ2.zip
05-05-2000  00:00:00              1550521 Ftpd106.zip
02-07-2001  00:00:00                35459 hmailbox.zip
05-05-2000  00:00:00                55463 HRxMail.zip
05-05-2000  00:00:00                34950 HRxPass.zip
05-05-2000  00:00:00              1320184 inetmail-1.2.1.pro.zip
05-05-2000  00:00:00              1218483 inetmail-1.2.1.zip
05-05-2000  00:00:00              1615428 inetmail-1.3.0.pro.zip
05-05-2000  00:00:00              1425567 inetmail-1.3.0.zip
01-09-2001  00:00:00              1629145 inetmail-1.3.18.zip
02-07-2001  00:00:00              1625314 inetmail-1.3.18a.zip
05-05-2000  00:00:00              1407069 inetmail-1.5.0.pro.zip
09-27-2000  00:00:00              1470338 inetmail-1.5.31.pro.zip
01-04-2001  00:00:00              1583435 inetmail-1.5.32.pro.zip
05-05-2000  00:00:00              1429364 inetmail-1.5.5.pro.zip
05-05-2000  00:00:00              1471373 inetmail-1.5.6.pro.zip
02-08-2001  00:00:00       <DIR>          misc
05-05-2000  00:00:00                30913 MXLookup.zip
05-02-2002  00:00:00       <DIR>          perseus
02-08-2001  00:00:00       <DIR>          rfcs
02-08-2001  00:00:00       <DIR>          scripts
02-08-2001  00:00:00       <DIR>          tcpip
05-05-2000  00:00:00       <DIR>          testcase
02-07-2001  00:00:00       <DIR>          tools
